src/event.c

```diff
@@ -208,13 +208,17 @@ int event_encode_dict(struct odict *od, struct ua *ua, enum ua_event ev,
 			err |= odict_entry_add(od, "diverteruri", ODICT_STRING,
 					       call_diverteruri(call));
 
+		err |= odict_entry_add(od, "onhold", ODICT_BOOL,
+				call_is_onhold(call));
+
```

Comment:
This is only the local on-hold flag. The application should know that it put the call on hold locally. Is this worth to be put to the event?
Should we add it only if `call_is_onhold()`?

Reply:
For me it is important to know if a call has been put on hold, regardless if it was triggered by the local client or the remote party. I am using the mqtt module to get updates about the call states and HOLD is something I am interested in.

If the remote party sets the call to HOLD, the call_is_onhold is not updated?